testing/a11y/cypress.config.js

```diff
@@ -20,6 +20,7 @@ export default defineConfig({
     },
     env: { axeCorePath },
     video: false,
+    numTestsKeptInMemory: 205,
```

Comment:
is this change needed?

Reply:
It keeps snapshots from every component test; by default, it only keeps fifty of them. Without this change, you would only be able to view the snapshots from the last fifty examples tested.

Reply:
then lets pick a higher number and guard it under a CI check (there is limited memory in CI)


something like this might work (untested)
```suggestion
    numTestsKeptInMemory: process.env.CI ? 500 : undefined,
```

Reply:
whoops, the condition is flipped. should be:

```
    numTestsKeptInMemory: !process.env.CI ? 500 : undefined,
```

("when not CI, use 500, otherwise leave it undefined")

make sure to test that it works as expected locally

Reply:
It worked locally, pushed